src/test/java/org/javarosa/core/model/utils/test/DateUtilsSCTOTests.java

```diff
@@ -77,27 +60,25 @@ public void testParseTime() {
     // - We're effectively binding all times to the EPOCH date
```

Comment:
Is this comment still accurate? Aren't all times now bound to whatever the current time of the device is?